client/my-sites/plans/jetpack-plans/build-checkout-url.ts

```diff
@@ -39,7 +43,11 @@ export default function buildCheckoutURL(
 			? 'http://calypso.localhost:3000'
 			: 'https://wordpress.com';
 
-	if ( ! siteSlug && config.isEnabled( 'jetpack/siteless-checkout' ) ) {
+	if (
+		! siteSlug &&
+		config.isEnabled( 'jetpack/siteless-checkout' ) &&
+		! [ PRODUCT_JETPACK_SEARCH, PRODUCT_JETPACK_SEARCH_MONTHLY ].includes( productsString )
+	) {
```

Comment:
Jetpack's Pricing page should lead users to Jetpack Connect when they click on Jetpack Search product card.

Reply:
Could `JETPACK_SEARCH_PRODUCTS` be used here as well?

Reply:
Yes, we could use that constant there. I didn't use it because that function is used only on the Jetpack's version of the pricing page where users can't choose the WPCOM's version of Search.